scripts/setup_gae.sh

```diff
@@ -31,4 +31,22 @@ export PYTHONPATH=.:$COVERAGE_HOME:$GOOGLE_APP_ENGINE_HOME:$GOOGLE_APP_ENGINE_HO
 # Delete old *.pyc files
 find . -iname "*.pyc" -exec rm -f {} \;
 
+# Moved from install .travis.yml.
```

Comment:
(If you meant "Moved from setup.sh", then I get it, but you don't really need the comment. Future developers wouldn't know that this code was in setup.sh before.)